src/quotas.md

```diff
@@ -21,6 +21,10 @@ period (daily, monthly, yearly). Account quotas may be affected by the verificat
 
 {% endpoint GET https://service.centrapay.com/api/accounts/{accountId}/quotas %}
 
+Quota limits and usages for the current intervals will can be retrieved. Ie,
```

Comment:
typo should be `Quota limits and usages for the current intervals can be retrieved` I think